src/server/console/minimal_console_services.cpp

```diff
@@ -100,7 +100,7 @@ std::future<std::unique_ptr<mir::Device>> mir::MinimalConsoleServices::acquire_d
     auto const devnode = [](auto const& fd) {
         using namespace boost::iostreams;
         char line_buffer[1024];
-        stream<file_descriptor_source> uevent{fd, file_descriptor_flags::never_close_handle};
+        stream<file_descriptor_source> uevent{(int)fd, file_descriptor_flags::never_close_handle};
```

Comment:
`Fd` has an implicit conversion to `int`, so I'm surprised this is needed. But it should not be a c-style cast.

Reply:
Switched to `static_cast<int>(fd)`